### PR TITLE
RelNotes 8 Draft

### PR DESCRIPTION
**Pull request by rido-min**

### CODE REVIEW

File: articles/release-notes/release-update-8.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+     ms.service="na"
+     ms.devlang="javascript"
+     ms.topic="article"
+     ms.tgt_pltfrm="mobile-multiple"
+     ms.workload="na"
+     ms.date="03/08/2016"
+     ms.author="rmpablos"/>
+
+#**Update 8 - Visual Studio Tools for Apache Cordova**
+Update 8 corresponds to Visual Studio Tools for Apache Cordova version number 14.XXXX.X
```
Comment (jmatthiesen) [inline articles/release-notes/release-update-8.md]: Do we have a version number we can enter on this row, yet?

**Comment by kirupa** [comment]
There was an error while validating job c060f97d-0079-4291-a976-e5867d2cd8f5 <br/><hr/><b>Message:</b><pre>

Result status: Failed

There was a problem accessing Github repository 'https://github.com/Microsoft/cordova-docs.git'.

</pre><b>Details:</b><pre>Microsoft.LightweightJobEngine.Core.Language.ContextCommandBase+ContextCommandException: 

Result status: Failed

There was a problem accessing Github repository 'https://github.com/Microsoft/cordova-docs.git'.

 ---> System.AggregateException: One or more errors occurred. ---> System.ArgumentException: Error clonning repository ---> LibGit2Sharp.LibGit2SharpException: Failed to receive response: The server returned an invalid or unrecognized response

   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_remote_fetch(RemoteSafeHandle remote, Signature signature, String logMessage)
   at LibGit2Sharp.Network.DoFetch(RemoteSafeHandle remoteHandle, FetchOptions options, Signature signature, String logMessage)
   at LibGit2Sharp.Network.Fetch(String url, IEnumerable`1 refspecs, FetchOptions options, Signature signature, String logMessage)
   at Microsoft.DPS.Services.LibGit2SharpSourceControlProvider.ExecuteFetch(String url, Repository repo, IEnumerable`1 refspecs)
   at Microsoft.DPS.Services.LibGit2SharpSourceControlProvider.<>c__DisplayClass34.<CloneRepository>b__2c()
   at Microsoft.Practices.EnterpriseLibrary.TransientFaultHandling.RetryPolicy.<>c__DisplayClass1.<ExecuteAction>b__0()
   at Microsoft.Practices.EnterpriseLibrary.TransientFaultHandling.RetryPolicy.ExecuteAction[TResult](Func`1 func)
   at Microsoft.Practices.EnterpriseLibrary.TransientFaultHandling.RetryPolicy.ExecuteAction(Action action)
   at Microsoft.DPS.Services.LibGit2SharpSourceControlProvider.CloneRepository(String url, IDictionary`2 branches)
   --- End of inner exception stack trace ---
   at Microsoft.DPS.Services.LibGit2SharpSourceControlProvider.CloneRepository(String url, IDictionary`2 branches)
   at Microsoft.DPS.Services.LibGit2SharpSourceControlProvider.CloneRepository()
   at Microsoft.DPS.Services.LibGit2SharpSourceControlProvider.GetVersionAsync(String path)
   at Microsoft.DPS.Commands.IdentifyDocumentsToPublish.CheckExternalServicesCommand.Do(IDictionary`2 context)
   --- End of inner exception stack trace ---
   --- End of inner exception stack trace ---
   at Microsoft.LightweightJobEngine.Core.Language.ContextCommandBase.ThrowIfErrorsLogged()
   at Microsoft.DPS.Commands.IdentifyDocumentsToPublish.CheckExternalServicesCommand.Do(IDictionary`2 context)
   at Microsoft.LightweightJobEngine.Core.Language.JobWorkflow`1.RunCommands(Guid jobId, IDictionary`2 context, CancellationTokenSource cancelSignal)</pre>

File: articles/release-notes/release-update-8.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+     ms.author="rmpablos"/>
+
+#**Update 8 - Visual Studio Tools for Apache Cordova**
+Update 8 corresponds to Visual Studio Tools for Apache Cordova version number 14.XXXX.X 
+
+## Setup Instructions
+The most common way to get this update will be the Notification Icon in VS and the Tools & Extensions Updates,
+however from this update we are offering also a standalone installer that you can find here:
+
+[Visual Studio Tools for Apache Cordova Update 8 Download](http://go.microsoft.com/fwlink/?LinkId=XXXX)
```
Comment (jmatthiesen) [inline articles/release-notes/release-update-8.md]: Can we update this download link with a real LinkId?

File: articles/release-notes/release-update-8.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Visual Studio offers a list of most common used plugins. We are replacing the `com.microsoft.azure-mobile-services` with the new `cordova-plugin-ms-azure-mobile-apps`. 
+We also added more popular plugins:
+- `cordova-plugin-ms-azure-mobile-apps`
+- `cordova-plugin-hockeyapp`
+- `cordova-plugin-code-push`
+- `cordova-plugin-bluetoothle`
+- `phonegap-plugin-push`
+- `cordova-plugin-ms-azure-mobile-engagement`
+
+## Bugs fixed in this release
```
Comment (jmatthiesen) [inline articles/release-notes/release-update-8.md]: Any bugs to call out?

File: articles/release-notes/release-update-8.md (hunk 1 above)
Comment (rido-min) [inline articles/release-notes/release-update-8.md]: done

File: articles/release-notes/release-update-8.md (hunk 2 above)
Comment (rido-min) [inline articles/release-notes/release-update-8.md]: done

File: articles/release-notes/release-update-8.md (hunk 3 above)
Comment (rido-min) [inline articles/release-notes/release-update-8.md]: asking to DEV

**Pull request merged**